src/cmd/src/error.rs

```diff
@@ -415,6 +423,7 @@ impl ErrorExt for Error {
             }
             Error::MetaClientInit { source, .. } => source.status_code(),
             Error::TonicTransport { .. } => StatusCode::Internal,
+            Error::SchemaNotFound { .. } => StatusCode::Unexpected,
```

Comment:
We have a `DatabaseNotFound = 4004` for this case

```suggestion
            Error::SchemaNotFound { .. } => StatusCode::DatabaseNotFound,
```